src/grid.js

```diff
@@ -58,6 +58,22 @@ var $$parseSort = function(ids) {
   });
 };
 
+var SCROLLBAR_WIDTH = 16;
```

Comment:
Scrollbar width is browser dependent, we can probably [compute it dynamically](http://davidwalsh.name/detect-scrollbar-width).